tests/integration/components/speaker-list-test.js

```diff
@@ -0,0 +1,25 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('speaker-list', 'Integration | Component | speaker list', {
```

Comment:
See other component tests and use the custom `moduleForComponent` method instead of the one from `ember-qunit`